modules/installation-configuration-parameters.adoc

```diff
@@ -2165,37 +2115,10 @@ ifeval::["{context}" == "installing-openstack-user-sr-iov-kuryr"]
 :!osp:
 :!osp-kuryr:
 endif::[]
-ifeval::["{context}" == "installing-vsphere-installer-provisioned-customizations"]
-:!vsphere:
-endif::[]
-ifeval::["{context}" == "installing-vsphere-installer-provisioned-network-customizations"]
-:!vsphere:
-endif::[]
-ifeval::["{context}" == "installing-vmc-customizations"]
-:!vmc:
-endif::[]
-ifeval::["{context}" == "installing-vmc-network-customizations"]
-:!vmc:
-endif::[]
-ifeval::["{context}" == "installing-restricted-networks-vmc"]
-:!vmc:
-endif::[]
```

Comment:
Worth noting that while this work is for IBM Power VS, I am now just noticing that https://github.com/openshift/openshift-docs/pull/60438, inadvertently added these directives back to the module. Thes vSphere directives are no longer used and were originally removed with  https://github.com/openshift/openshift-docs/pull/61165. The VMC directives are no longer used and were originally removed with  https://github.com/openshift/openshift-docs/pull/60614